.github/workflows/integration_test.yaml

```diff
@@ -244,6 +244,8 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: ${{ inputs.provider }}
+      - name: Copy kube-config file
+        run: sudo microk8s config > ~/.kube/config
```

Comment:
Is it worth adding an integration test that checks for this failure?

Reply:
That code is already exercised